Gruntfile.js

```diff
@@ -1,136 +1,261 @@
+      main: {
+        'dest': 't3SphinxThemeRtd/static/js/modernizr.min.js',
+        'options': {
+          'options': [
+            'domPrefixes',
+            'prefixes',
+            'addTest',
+            'hasEvent',
+            'mq',
+            'prefixedCSSValue',
+            'testAllProps',
+            'testProp',
+            'testStyles',
+            'setClasses'
+          ],
+          'feature-detects': [
+            'custom-elements',
+            'history',
+            'pointerevents',
+            'postmessage',
+            'webgl',
+            'websockets',
+            'css/animations',
+            'css/columns',
+            'css/flexbox',
+            'elem/picture',
+            'img/sizes',
+            'img/srcset',
+            'workers/webworkers'
+          ]
+        }
+      }
+    },
+
+    // uglify
+    uglify: {
+      options: {
+        output: {
+          comments: false
         }
       },
-      build: {
-        options: {
-          outputStyle: 'compressed'
-        },
+      autocomplete: {
+        src: 'node_modules/autocompleter/autocomplete.js',
+        dest: 't3SphinxThemeRtd/static/js/autocomplete.min.js'
+      },
+      underscore: {
+        src: 'node_modules/underscore/underscore.js',
+        dest: 't3SphinxThemeRtd/static/js/underscore.min.js'
+      },
+      modernizr: {
+        src: 't3SphinxThemeRtd/static/js/modernizr.min.js',
+        dest: 't3SphinxThemeRtd/static/js/modernizr.min.js'
+      },
+      // SOURCE: https://github.com/sphinx-doc/sphinx/blob/master/sphinx/themes/basic/static/doctools.js
+      doctools: {
+        src: 'js/doctools.js',
+        dest: 't3SphinxThemeRtd/static/js/doctools.min.js'
+      },
+      // SOURCE: https://github.com/sphinx-doc/sphinx/blob/master/sphinx/themes/basic/static/searchtools.js
+      searchtools: {
+        src: 'js/searchtools.js',
+        dest: 't3SphinxThemeRtd/static/js/searchtools.min.js'
+      },
+      theme: {
+        src: 'js/theme.js',
+        dest: 't3SphinxThemeRtd/static/js/theme.min.js'
+      },
+    },
+
+    // remmove sourcemaps of used dist files
+    removesourcemap: {
+      contrib: {
         files: {
-          't3SphinxThemeRtd/static/css/badge_only.css': 'sass/badge_only.sass',
-          't3SphinxThemeRtd/static/css/t3more.css': 'sass/t3more.sass',
-          't3SphinxThemeRtd/static/css/theme.css': 'sass/theme.sass',
-          't3SphinxThemeRtd/static/css/theme-no-fonts.css': 'sass/theme-no-fonts.sass'
+          't3SphinxThemeRtd/static/js/jquery.min.js': 't3SphinxThemeRtd/static/js/jquery.min.js',
+          't3SphinxThemeRtd/static/js/bootstrap.min.js': 't3SphinxThemeRtd/static/js/bootstrap.min.js',
+          't3SphinxThemeRtd/static/js/popper.min.js': 't3SphinxThemeRtd/static/js/popper.min.js'
         }
       }
     },
 
+    // exec
     exec: {
       build_sphinx: {
-        cmd: './my-sphinx-build.sh demo_docs/source demo_docs/build'
+        command: 'docker run --rm '
```

Comment:
On Linux Docker creates files that belong to the `root` user. So only the first build succeeds, because the user is not allowed to clean those file. (S)he needs to `sudo chown -R $(id -u):$(id -g) build/` between builds. For Linux users the correct line would be `command: 'docker run --rm --user=$(id -u):$(id -g) '` This is fine with Macs too, but doesn't work for Windows users. I don't know if Windows users are using this `yarn` and `grunt` stuff. If we have to account for Windows users, I'd leave the line as it is. Otherwise I recommend to add the `--user=$(id -u):$(id -g)` option.

Reply:
I am Windows User 😂

Reply:
@marble please check the bugfix I provided, not sure of we also need to set the user rights on macs too. https://github.com/TYPO3-Documentation/t3SphinxThemeRtd/pull/140/commits/06986e8ece9ef47e0f3d8f1de1f3bceecb2ac326

Reply:
AFAIR Macs don't core. But I know that `$(id -u)…` works for them. The thing is, that Windows users don't need that.

Reply:
Ok, I´ve added a check for Linux users to add the stuff for now. I will try to find someone with a mac to also check if it's necessary here.